### DISCUSSION TITLE
Is 1.0.0 will be the first release?

### DISCUSSION THREAD

**Discussion by kakawait** [excalideck/excalideck#38] [General]
I really like the look & feel of Excalidraw. And Excalideck is a great idea to give possibility to re-use that concept for slides.

But I'm wondering, do you plan to release some intermediate (even BETA or other flag) before 1.0.0?

Regarding the milestone https://github.com/excalideck/excalideck/milestone/2 the quantity of work isn't trivial.

**Comment by pscanf** [comment]
Hey @kakawait, thanks for asking!

Yes, I do plan to have intermediate releases. CI/CD is set up so that every merge to master releases to production, so in general new releases will come out as I work on the product.

The `1.0.0` milestone is more of an artificial goal to decide what to prioritise. Right now Excalideck is in an **"unpolished MVP state"**: the most important features are there, they work, though not always with an optimal UX, and sometimes with bugs and other defects. The plan is now to go towards **"state `1.0.0`"**, where most bugs will be fixed and the UX will generally be more polished. After that I'll start working on adding other more substantial features.
